scenes/test_player.gd

```diff
@@ -0,0 +1,28 @@
+extends CharacterBody2D
+
+
+const SPEED = 300.0
```

Comment:
Know this is a test_player, but do we want to leave SPEED as as a const, or have it be a var so can potentially modify it through the lifetime of the game?

Reply:
yeah I'll export speed and maybe even gravity as editable variables once we have a real player script